doc/builders/trivial-builders.section.md

```diff
@@ -0,0 +1,48 @@
+# Trivial builders {#chap-trivial-builders}
+
+Nixpkgs provides a couple of functions that help with building derivations. The most important one, `stdenv.mkDerivation`, has already been documented above. The following functions wrap `stdenv.mkDerivation`, making it easier to use in certain cases.
+
+
+- `runCommand` {#trivial-builder-runCommand}
+        This takes three arguments, `name`, `env`, and `buildCommand`. `name` is just the name that Nix will append to the store path in the same way that `stdenv.mkDerivation` uses its `name` attribute. `env` is an attribute set specifying environment variables that will be set for this derivation. These attributes are then passed to the wrapped `stdenv.mkDerivation`. `buildCommand` specifies the commands that will be run to create this derivation. Note that you will need to create `$out` for Nix to register the command as successful.
+
+        An example of using `runCommand` is provided below.
+
+```nix
+(import <nixpkgs> {}).runCommand "my-example" {} ''
+  echo My example command is running
+
+  mkdir $out
+
+  echo I can write data to the Nix store > $out/message
+
+  echo I can also run basic commands like:
+
+  echo ls
+  ls
+
+  echo whoami
+  whoami
+
+  echo date
+  date
+''
+```
+
+- `runCommandCC` {#trivial-builder-runCommandCC}
```

Comment:
You would need to use the [bracketed span](https://pandoc.org/MANUAL.html#extension-bracketed_spans) syntax to add ids.

```suggestion
- [`runCommandCC`]{#trivial-builder-runCommandCC}
```

Reply:
Alternately, you cold convert the `varlistentry` definitions to headings. This would also allow us to get rid of the ugly indentation.

Reply:
@jtojnar something like that?

```markdown
# `runCommandCC`{#trivial-builder-runCommandCC}
```

Reply:
Yes, only use lower level headings – you can only have a single level 1 heading in a section.